stripe/util.py

```diff
@@ -58,3 +72,38 @@ def utf8(value):
 def is_appengine_dev():
     return ('APPENGINE_RUNTIME' in os.environ and
             'Dev' in os.environ.get('SERVER_SOFTWARE', ''))
+
+
+def _console_log_level():
+    if stripe.log in ['debug', 'info']:
+        return stripe.log
+    elif STRIPE_LOG in ['debug', 'info']:
+        return STRIPE_LOG
+    else:
+        return None
+
+
+def log_debug(message, *args, **kwargs):
+    if _console_log_level() == 'debug':
+        print(message.format(*args, **kwargs))
+    logger.debug(message.format(*args, **kwargs))
+
+
+def log_info(message, *args, **kwargs):
+    if _console_log_level() in ['debug', 'info']:
+        print(message.format(*args, **kwargs))
+    logger.info(message.format(*args, **kwargs))
+
+def _test_or_live_environment():
+    if stripe.api_key is None:
+        return
+    match = re.match(r'sk_(live|test)_', stripe.api_key)
```

Comment:
Ok, I think then I'll make sure it can deal with older keys without barfing, and just do what it does in the case of no api-key and default to test.